src/buildHTML.js

```diff
@@ -58,9 +55,11 @@ const styleMap = {
  * nodes. documentFragments are flattened into their contents, so the
  * returned list contains no fragments. `isRealGroup` is true if `expression`
  * is a real group (no atoms will be added on either side), as opposed to
- * a partial group (e.g. one created by \color).
+ * a partial group (e.g. one created by \color). `surrounding` is an array
+ * consisting type of nodes that will be added to the left and right.
  */
-export const buildExpression = function(expression, options, isRealGroup) {
+export const buildExpression = function(expression, options, isRealGroup,
+        surrounding = []) {
```

Comment:
Maybe make this `surrounding = [null, null]`.  I think eventually we'll want to enforce that this is a 2-tuple with `flow`.